src/components/Box.vue

```diff
@@ -1,13 +1,35 @@
 <template>
-  <div v-bind="$attrs" v-on="$listeners" class="box rounded-borders">
+  <div v-bind="$attrs" v-on="$listeners" class="box rounded-borders q-px-md q-py-lg" :class="boxClass">
     <slot />
   </div>
 </template>
 
+<script>
+export default {
+  props: {
+    formMode: {
+      type: Boolean
+    }
+  },
+
+  computed: {
+    boxClass () {
+      return this.formMode ? 'box__border' : 'box__shadow'
+    }
```

Comment:
Eu achei pouco semântico esse "formMode", não poderiam ter as opções como `border` e `shadow`?

Reply:
Mas é justamente para quando for usar em formulario.

Reply:
É para ter comportamento de formuario